renku/cli/dataset.py

```diff
@@ -179,6 +179,30 @@
 
     $ renku dataset add my-dataset -e /path/to/external/file
 
+Moving/renaming files
+
+It is possible to move files between datasets or rename a single file.
+``renku dataset mv`` command accepts a source and a destination dataset
+and moves all files that match a specific pattern from source to destination:
+
+.. code-block:: console
+
+    $ renku dataset mv src dst -I '*csv'
+
+This command moves all CSV files from ``src`` dataset to ``dst``. If no
+pattern is given then all files are moved. Note that move command works only
+with files and it moves all files to the root of the destination's data
+directory. You can pass a ``--destination`` or ``-d`` flag to specify a
+different directory as the destination.
+
+If only one file is moved and a destination is passed, then the file will be
+renamed. The following command renames ``data.csv`` to ``old-data.csv`` in the
+same dataset:
+
+.. code-block:: console
+
+    $ renku dataset mv my-dataset my-dataset -I data.csv -d old-data.csv
```

Comment:
This is a helpful example because it's a little counterintuitive that to rename a file you have to pass in the same dataset as src and dst. Could you add this to the text that is shown with `renku dataset mv --help`?

Reply:
I've fixed this; now, if one doesn't specify a destination dataset then the source dataset is used as destination.